src/transformers/utils/hub.py

```diff
@@ -428,21 +429,26 @@ def cached_file(
             use_auth_token=use_auth_token,
             local_files_only=local_files_only,
         )
-
-    except RepositoryNotFoundError:
+    except GatedRepoError as e:
+        raise EnvironmentError(
+            "You are trying to access a gated repo.\nMake sure to request access at "
+            f"https://huggingface.co/{path_or_repo_id} and pass a token having permission to this repo with "
+            "`use_auth_token` or log in with `huggingface-cli login`."
```

Comment:
```suggestion
            "You are trying to access a gated repo.\nMake sure to request access at "
            f"https://huggingface.co/{path_or_repo_id} and pass a token having permission to this repo either "
            "by logging in with `huggingface-cli login` or with `token=<your_token>`."
```
We have started deprecating `use_auth_token` to go with `huggingface-hub`, so let's use `token` here.

Reply:
Ah, good to know! I wasn't sure to I reused the current wording. Will make the change :)